pocs/tests/test_observatory.py

```diff
@@ -227,6 +282,23 @@ def test_get_observation(observatory):
     assert observatory.current_observation == observation
 
 
+def test_get_observation_no_scheduler(observatory):
+    obs = observatory
+    obs.scheduler = None
+    assert obs.get_observation() is None
+
+
+def test_cleanup_observations_no_scheduler(observatory):
+    obs = observatory
+    obs.scheduler = None
+    assert obs.cleanup_observations() is None
+
+
+def test_cleanup_observations_keep_jpgs(observatory):
```

Comment:
This is unrelated to this PR and I'm not sure is testing what we want. Let's just remove for now.